ckanext/ytp/request/controller.py

```diff
@@ -19,6 +19,7 @@ def _list_organizations(context, errors=None, error_summary=None):
         data_dict['all_fields'] = True
         data_dict['groups'] = []
         data_dict['type'] = 'organization'
+        data_dict['limit'] = 1000
```

Comment:
Why do we need to set this up here if it's already configured in ckan.ini?
Also if using constant it's better to put them to the top of the module and capize the var name:
```
LIST_ORGANIZATIONS_LIMIT = 1000
[...]
data_dict['limit'] = LIST_ORGANIZATIONS_LIMIT
```

Reply:
I really don't like the way you get silent clipping of the list.  The JSON response should have some indication of pagination or something, but it doesn't.  I've put this here so that you can see how the limit can be specified in two places: the config and the action params.  It's not needed from a functional perspective.  It's purely for the sake of readability really. Happy for it to be removed. 

If we do keep it, on second thoughts, it probably should be dynamically set to the adx_config.ini value anyway, since this was the intention at the time of writing.